### PR TITLE
Add theme font sizes

### LINKED ISSUE #3: Update typography settings
Update the typography settings so that they match the [desktop](https://www.figma.com/file/nf4oV51c1JBxJKXshMe7cd/New-Sensei-Theme?node-id=20020042%3A412) and [mobile](https://www.figma.com/file/nf4oV51c1JBxJKXshMe7cd/New-Sensei-Theme?node-id=20020042%3A427) designs. Example PRs [here](https://github.com/Automattic/themes/pull/6479), [here](https://github.com/Automattic/themes/pull/6487) and [here](https://github.com/Automattic/themes/pull/6571).

## Desktop
Screen Shot 2022-09-30 at 3 12 13 PM

## Mobile
Screen Shot 2022-09-30 at 3 12 40 PM

#### Scenario: Typography settings
**Given** The Sensei theme is activated
**When** I view a page containing `h1` - `h4` headings and paragraph blocks
**Then** the typography looks similar to the design on both desktop and mobile.

### PR DESCRIPTION
**Pull request by gabrielcaires**
# Introduced changes
* Add all theme font sizes

Screen Shot 2022-10-05 at 18 16 08

## Testing instructions:
* Create a page
* Copy and past the following post content

```
<!-- wp:post-date /-->

<!-- wp:heading {"level":1} -->
<h1>I am THE H1 TAG</h1>
<!-- /wp:heading -->

<!-- wp:heading -->
<h2>I am the H2 tag</h2>
<!-- /wp:heading -->

<!-- wp:heading {"level":3} -->
<h3>I am the H3</h3>
<!-- /wp:heading -->

<!-- wp:heading {"level":4} -->
<h4>I am the H4</h4>
<!-- /wp:heading -->

<!-- wp:heading {"level":4} -->
<h4>I am the h5</h4>
<!-- /wp:heading -->

<!-- wp:heading {"level":6} -->
<h6>I am the h6</h6>
<!-- /wp:heading -->

<!-- wp:paragraph -->
<p>This blog is run by Professor Jean Biche, Chair of Art History at Sacré-Cœur Université Paris. He joined the Department of Art in 2009 moving from the University of Notredame where he was a Professor of Global History and Culture.</p>
<!-- /wp:paragraph -->

<!-- wp:heading {"level":1,"fontSize":"large"} -->
<h1 class="has-large-font-size">group/section titles</h1>
<!-- /wp:heading -->

<!-- wp:heading {"level":1,"fontSize":"x-large"} -->
<h1 class="has-x-large-font-size">posts titles on grid</h1>
<!-- /wp:heading -->

<!-- wp:heading {"level":1,"fontSize":"xx-large"} -->
<h1 class="has-xx-large-font-size">POST TITLES ON LISTS</h1>
<!-- /wp:heading -->

<!-- wp:heading {"level":1,"fontSize":"4x-large"} -->
<h1 class="has-4-x-large-font-size">DISPLAY TITLE</h1>
<!-- /wp:heading -->
```

### CODE REVIEW

**Comment by LavaGolem** [comment]
Here are the issues I found: 
**H1**
1. It is the smallest: 
Edit: 
image
Preview: 

image

**Title**
Title has wrong font in preview (in edit it is a correct font but small)
Edit:
image

Preview: 
image

**Review by LavaGolem — CHANGES_REQUESTED** [review]
Here are the issues I found: 
**H1**
1. It is the smallest: 
Edit: 
image
Preview: 

image

**Title**
Title has wrong font in preview (in edit it is a correct font but small)
Edit:
image

Preview: 
image

**Review by LavaGolem — COMMENTED** [review]
The **paragraph** text is too small 
image

File: theme.json [hunk 1]
```diff
@@ -280,7 +300,7 @@
 			"h1": {
 				"typography": {
           "fontFamily": "var(--wp--preset--font-family--league-lothic)",
-					"fontSize": "clamp(3rem, 5vw + 1.5rem, 6.029rem)",
+					"fontSize": "var(--wp--preset--font-size--3x-large)",
```
Comment (LavaGolem) [inline theme.json]: this one is not correct. It should be: 

``` "fontSize": "var(--wp--preset--font-size--3-x-large)", ```

This fixes the h1 size

Comment (gabrielcaires) [reply theme.json]: Fixed here https://github.com/Automattic/sensei-theme/pull/19/commits/7098138269d2b636fc1b2658419f79765aa47bc8

**Review by LavaGolem — APPROVED** [review]

**Pull request merged**